public/angular/js/controllers/sources/index.js

```diff
@@ -19,11 +19,12 @@ angular.module('Aggie')
     };
 
     $scope.target = function(source) {
-      return source.media == 'twitter' || source.media == 'smsgh' || source.media == 'whatsapp' ? source.keywords : source.url;
+      return source.media == 'twitter' || source.media == 'smsgh' || source.media == 'whatsapp' || source.media == 'crowdtangle' ? source.keywords : source.url;
     };
 
     $scope.sourceClass = function(source) {
-      var mediaOptions = ['twitter', 'facebook', 'rss', 'elmo', 'smsgh', 'whatsapp'];
+      var mediaOptions = ['twitter', 'crowdtangle'];
```

Comment:
@TheDanieLNK Why are these the only available `mediaOptions`?

Reply:
Yes. I agree we should have a Myanmar-specific branch. I will re-enter the previous options and comment them out. No pressures though since most of these sources (SMSGh, RSS) won't be use anytime soon.